lib/helper/WebDriver.js

```diff
@@ -1581,6 +1581,41 @@ class WebDriver extends Helper {
     await browser.buttonUp(0);
   }
 
+  /**
+   * Get all Window Handles.
+   * Useful for referencing a specific handle when calling `I.switchToWindow(handle)`
+   *
+   * ```js
+   * const handles = await I.getWindowHandles();
+   * ```
+   */
+  async getWindowHandles() {
```

Comment:
Ok, I agree to have this but let's get to standard naming:

```js
const windows = await I.grabAllWindowHandles();
```

Reply:
Ok, I agree - pushing new changes now.